protostar/commands/test/testing_summary.py

```diff
@@ -183,7 +188,9 @@ def _format_slow_test_cases_list(
         local_log_color_provider: LogColorProvider = log_color_provider,
     ) -> str:
 
-        slowest_test_cases = self._get_slowest_test_cases_list(self.failed + self.passed, count)  # type: ignore
+        slowest_test_cases = self._get_slowest_test_cases_list(
+            self.failed + self.passed + self.broken, count  # type: ignore
```

Comment:
I think `type: ignore` can be now removed.

Reply:
I couldn't find a way to make it work without including some ugly code